### DISCUSSION TITLE
Changing the font size of the tree view

### DISCUSSION THREAD

**Discussion by mramendi** [pulsar-edit/.github#198] [General]
I would like to increase the font size in the tree view, or else just make it the same as the font size in the editor.

I did not find any such option in the menus. When I googled how to do it in Atom, I found an explanation about using the `styles.less` file in `~/.atom/` ; however, there is no `styles.less` file in `~/.pulsar/` .

So how do I do this?

**Comment by asiloisad** [comment]
If you are use `one-light-ui` or `one-dark-ui` themes, then go Settings -> Themes -> setting button next to UI Theme list -> Font Size. It change all UI elements. You can change only tree-view font-size, then open File -> Stylesheet... and paste below code

```less
.tree-view {
  font-size: 15px;
}
```

**Comment by confused-Techie** [comment]
@mramendi Also you can just make a `styles.less` file if there isn't one, Pulsar will automatically pick it up on launch or refresh. But like mentioned above, there's an option within Pulsar to open up that file, or the font options in settings will work as well.
